pkg/controller/test/actions.go

```diff
@@ -84,5 +84,8 @@ func (a *action) Matches(act coretesting.Action) error {
 		return nil
 	}
 
+	fmt.Printf("EXP:%+v\n", objExp.GetObject())
+	fmt.Printf("ACT:%+v\n", objExp.GetObject())
```

Comment:
Do we need these changes? If so, can you tidy up the messages? Looks like it might have been your testing/debugging changes 😄